docs/release.md

```diff
@@ -1,11 +1,16 @@
 # How to create a release
 
-Push a git tag with the version you wish to release.
+- Ensure [CHANGELOG.md](/CHANGELOG.md) is up to date.
+- Add heading with version and release date. e.g. `## 0.1.0 2022-06-21`
+- Push a git tag with the version you wish to release.
 
 ```
-git tag -a v0.1.0 -m "Initial release"
+git tag -a v0.1.0 -m "v0.1.0"
 ```
 
+- Once the release is created edit the description to add the changelog text.
+- Note: The release description will already have the commits added by GoReleaser.
```

Comment:
There are some manual steps here but I don't think they are too onerous and we don't create that many releases.

See https://github.com/thegreenwebfoundation/grid-intensity-go/releases/tag/v0.3.0 for the current release description created by GoReleaser with the commits.